tests/tests_suite_test.go

```diff
@@ -44,8 +44,12 @@ var (
 )
 
 func getDir() string {
-	_, filename, _, _ := runtime.Caller(1)
-	return path.Dir(filename)
+	if os.Getenv("HOME") != "/root" {
```

Comment:
Testing for the presence of a `.dockerinit` (or `.dockerenv`) file is one way to know if you're inside a container.
```go
if _, err := os.Stat("/.dockerinit"); err == nil {
```